### PR TITLE
No account found

### PR DESCRIPTION
**Pull request by gguijarro-c-chwy**
Handle parsing error ESR link

Handle when the user taps on a ESR link but no account is active.

### CODE REVIEW

File: lib/ui/blocs/deeplink/deeplink_bloc.dart [hunk 1]
```diff
@@ -110,12 +117,22 @@ class DeeplinkBloc extends Bloc<DeeplinkEvent, DeeplinkState> {
   Future<void> _incomingESRLink(_IncomingESRLink event, Emitter<DeeplinkState> emit) async {
     final result = await _parseQRCodeUseCase.run(ParseESRLinkInput(esrLink: event.link));
     if (result.isValue) {
-      emit(
-        state.copyWith(
-          command: PageCommand.navigateToSignTransaction(result.asValue!.value),
-        ),
-      );
+      if (_authRepository.currentAuthStatus is Authenticated) {
```
Comment (gguijarro-c-chwy) [inline lib/ui/blocs/deeplink/deeplink_bloc.dart:120]: @n13  Hey nink, im not sure if this check is correct. Do we need the user to be authenticated before we can launch sign a transaction screen? 

Is so, then this check is correct

Comment (n13) [reply lib/ui/blocs/deeplink/deeplink_bloc.dart:120]: That is a good question. 

I think we should make this more generic and abstract it out to a "canSignTransaction()" method somewhere appropriate. 

In the Hypha wallet at the moment, we need to be authenticated

But for example in the seeds wallet we have an action that can be called un-authenticated, when we accept an invite link. The seeds wallet signs the invite link with a semi-public private key- meaning the private key is not a secret, it's in the repo - and the action it calls then creates an account for the user, so they're only authenticated later on. 

I will leave it up to you where to put this check - but it should take the ESR code as a parameter

The current implementation would simply do the check above - see if there's an authenticated user - and we can refine it later if needed.

**Comment by n13** [comment]
Good stuff

**Review by n13 — APPROVED** [review]

**Review by n13 — APPROVED** [review]

**Comment by n13** [comment]
LGTM

**Pull request merged**